pkg/pocket/core/generate.go

```diff
@@ -50,8 +49,12 @@ func (c *Core) beforeGenerate() error {
 		}
 		log.Infof("table %s generate", sql.SQLTable)
 
-		if c.cfg.Mode == "tiflash" {
-			c.createTiFlashTableReplica(sql, e)
+		if e.TiFlash {
+			if err := e.WaitTiFlashTableSync(sql.SQLTable); err != nil {
```

Comment:
Can we waiting for table sync together after all create table statements are executed?

Reply:
Good idea. Will update it tomorrow